### ISSUE TITLE
Add a @license command, separate from @copyright

### ISSUE THREAD

**Issue by eyalroz** [doxygen/doxygen#9786]
**Describe the bug**

Many users want/need to separate the text of their license from the copyright notice, using both a `@copyright` and a `@license` command.

**Expected behavior**
```
/** 
 * @file myfile.h
 *
 * @copyright (c) 20xx, 20xx, My Corp. All Rights Reserved.
 *
 * @license You are hereby granted a license to do whatever you want!
 */

and have each of these rendered nicely, separately.

Furthermore, it would be nice if it could support multiple paragraphs of license without special commands, but probably that's not possible.

**Version**
Looked up the lack of a @license command in the manual at: https://www.doxygen.nl/manual/commands.html , which should be fairly recent. "Verified" with a rather old version.

**Comment by albert-github** [comment]
Some extra info / thoughts:

- phpdocumeter (https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.license.pkg.html)
  > You may use the @license tag to document any element (include, page, class, function, define, method, variable)
  >
  > URL is any valid Uniform Resource Locator that points to the full text of a license. The description for the license is optional

  example
  ```
    /**
     * Displays <a href="http://opensource.org/licenses/gpl-license.php">GNU Public License</a>
     * @license http://opensource.org/licenses/gpl-license.php GNU Public License
     */
    class opensource_class {...}
  ```

**Comment by albert-github** [comment]
- single line / paragraph I think this is indeed correct and that a `@license` should be a paragraph
- regarding the link, normally I would say that all license text is shown verbatim, so without doxygen is interpreting it, though I think that when just a link is given the link should indeed be shown as a link.  The thing I wanted to accomplish with the `file` option is that a user doesn't have it include it into the source code but still can show the license verbatim, but I think you are right that this can also be accomplished by the `license` command as requested in this issue and using an `ALIASES` that uses the `\snippet` command, so probably no need for the `file` option. I defined it for convenience.
- multi paragraph, I fully agree though an  option like `multipar` is less confusing.


I don't get what you mean by the remark in the PS maybe you can give a small example.

**Comment by eyalroz** [comment]
> I don't get what you mean by the remark in the PS maybe you can give a small example.

I will, although it's out of scope. When I write, say:
```
/**
 * @brief My function. This does stuff.
 *
 * @note The quick brown fox jumps over the lazy dog. The quick brown fox jumps 
 * over the lazy dog. The quick brown fox jumps over the lazy dog. 
 *
 * Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
 * incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis 
 * nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
 *
 * @param[in] x a number
 **/
int foo(int x);
```
I expect the Lorem ipsum paragraph to be part of my note. It is only with the `@param` command that I expect the note to end and for something else to begin (and of course, there's a latent assumption here that `@param` takes a paragraph argument rather than being an intra-paragraph command like `@em`). However, doxygen produces:

> > **Note**
>    > The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog.
>
> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.

... which isn't what I wanted.

**Comment by albert-github** [comment]
I can imagine that this is not what you would like but it is as it is historically implemented and described in the documentation
>24.83 \note { text }
> Starts a paragraph where a note can be entered. The paragraph will be indented. The text of the paragraph has no
special internal structure. All visual enhancement commands may be used inside the paragraph. Multiple adjacent
\note commands will be joined into a single paragraph. Each note description will start on a new line. Alternatively,
one \note command may mention several notes. The \note command ends when a blank line or some other
sectioning command is encountered. See section \par for an example.


So placing `@note` before the `Lorem ...` will lead to:

![image](https://user-images.githubusercontent.com/5223533/212330625-ece6822e-73fa-440d-803d-0d91ceba7bfc.png)


So a little bit of effort but with the required result (I think)

**Comment by eyalroz** [comment]
@albert-github : I know that's how it works and is what's documented, but - I don't like it. When I write @note again, it reads like I'm making a second note.  You asked so I explained...

**Comment by ferdymercury** [comment]
I agree this is a much needed feature. Any plans for implementation? At least a simple version in the beginning. Thanks a lot!
